### PR TITLE
Adds the T-18 and T-12 for PFCs

### PR DESCRIPTION
**Pull request by Kuro-ai**
## About The Pull Request

Adds the T-18 Carbine and the T-12 Assault Rifle for PFCs

## Why It's Good For The Game

new guns woop woop

## Changelog
:cl:
add: T-18 and T-12 for PFCs
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -5. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by Teaika** [comment]
What about removing the barrel, making it part of the gun? I want a BC, ree

**Comment by Kuro-ai** [comment]
> What about removing the barrel, making it part of the gun? I want a BC, ree

The sprite didnt fit,this was the way to work around it

File: code/modules/projectiles/updated_projectiles/guns/rifles.dm [hunk 1]
```diff
@@ -37,53 +37,93 @@
 		return in_chamber ? (current_mag.current_rounds + 1) : current_mag.current_rounds
 
 //-------------------------------------------------------
-//M41A PULSE RIFLE
+//T-18 Carbine
 
 /obj/item/weapon/gun/rifle/standard_carabine
```
Comment (psykzz) [inline code/modules/projectiles/updated_projectiles/guns/rifles.dm]:
```suggestion
/obj/item/weapon/gun/rifle/standard_carbine
```

Can you also fix the occurences of this?

Comment (Kuro-ai) [reply code/modules/projectiles/updated_projectiles/guns/rifles.dm]: it has been done

**Comment by XSlayer300** [comment]
Hey Kuro! Do you mind of removing the M41A1 magazine boxes and its stock from vendors?

**Comment by Kuro-ai** [comment]
> Hey Kuro! Do you mind of removing the M41A1 magazine boxes and its stock from vendors?

Sure thing,no problem

**Comment by Rohesie** [comment]
Let us know when this is ready for merge.

**Comment by terranaut1** [comment]
The T12 is really loud. Tune it down a bit please.

**Comment by Rohesie** [comment]
Is the sound fine now?

**Comment by Kuro-ai** [comment]
Yes

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-9). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**